tests/service_reset.bats

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bats
+load test_helper
+
+setup() {
+  dokku "$PLUGIN_COMMAND_PREFIX:create" l
+}
+
+teardown() {
+  dokku --force "$PLUGIN_COMMAND_PREFIX:destroy" l
+}
+
+@test "($PLUGIN_COMMAND_PREFIX:reset) success with --force" {
+  run dokku --force "$PLUGIN_COMMAND_PREFIX:reset" l
+  assert_contains "${lines[*]}" "All l data deleted"
```

Comment:
These tests should also have `assert_success` or `assert_failure`.

Reply:
Fixed.

By the way, only `service_import.bats`, `service_link.bats` and `service_logs.bat` have `assert_success` or `assert_failure` after `assert_contains`.

Other `assert_contains` tests are missing them.